### DISCUSSION TITLE
Is there a way to prevent players from moving their homeblock to outposts?

### DISCUSSION THREAD

**Discussion by kevien12** [TownyAdvanced/Towny#7110] [Q&A] [answered]
I'm having an issue where players are moving their homeblock to an outpost to subvert the settings I have set for the merge command merge with towns thousands of blocks away from them. Is there a way to pervent players moving their homeblock to an outpost specifically or prevent them from merging with towns via outposts in general?

**Comment by LlmDl** [comment, marked as answer]
Not without using the API. An outpost townblock is known to be an outpost, but the townblocks connected to the outpost will not report as being part of an outpost.

I believe TownyProvinces does stop this in its own ways though.
